docs/extensions.md

```diff
@@ -2,18 +2,113 @@
 
 ## Creating extensions
 
-### Extension template
+OCDS extensions follow the structure of the [standard extension template](https://github.com/open-contracting/standard_extension_template). The template copies the structure of the core schema (except for _versioned-release-validation-schema.json_, see below) and allows you to add or update fields in those places where the extension should modify the schema using [JSON merge patch](https://tools.ietf.org/html/rfc7396).
 
-[Extension template](https://github.com/open-contracting/standard_extension_template)
+The core repository for OCDS has 4 [schema files](https://github.com/open-contracting/standard/tree/master/standard/schema):
+
+* release-schema.json
+* release-package-schema.json
+* record-package-schema.json
+* versioned-release-validation-schema.json
+
+_versioned-release-validation-schema.json_ is produced programmaticallly (see [Make validation schema](../deployment/standard-live#make-validation-schema) in the standard deployment section), which means that you don't have to author it manually when creating an extension.
+
+**TODO**: Mechanisn for extension authors to create _versioned-release-validation-schema.json_ in their extensions.
+
+All extensions must have an [extension.json](https://github.com/open-contracting/standard_extension_template/blob/master/extension.json), with `"name"` and `"description"` fields required.
+
+Extensions must include at least one schema file. In most cases, the extension will have a _release-schema.json_ with the minimal changes required to patch the schema, although there may be more marginal user cases requiring metadata patches for _release-package-schema.json_ and/or _record-package-schema.json_. Empty schema files should not be included in the extension.
+
+Repositories for [core extensions](http://standard.open-contracting.org/latest/en/extensions/#core-extensions) should have issue submissions disable by default and should direct developers to the [OCDS extension issues repository](https://github.com/open-contracting/ocds-extensions) to file issues. Best practice is to add that information to the _README.md_ file in every core extension using the following template:
+
+```none
+
+### Reporting issues
+
+By default, issues are disabled for individual OCDS core extensions. Instead, there is an [ocds extension repository]https://github.com/open-contracting/ocds-extensions) to gather issues for all extensions in a single place.
```

Comment:
I don't think we need to explain to everyone *why* we set things up the way we did. I think a simpler message works:

```
To report an issue about this extension, [create an issue here](https://github.com/open-contracting/ocds-extensions/issues), with the extension name in the title.
```